contrib/gocql/gocql/gocql_test.go

```diff
@@ -103,9 +103,9 @@ func TestChildWrapperSpan(t *testing.T) {
 	cluster := newCassandraCluster()
 	session, err := cluster.CreateSession()
 	assert.Nil(err)
-	q := session.Query("SELECT * from trace.person")
+	q := session.Query("SELECT * from trace.person").WithContext(ctx)
 	tq := WrapQuery(q, WithServiceName("TestServiceName"))
-	iter := tq.WithContext(ctx).Iter()
```

Comment:
Why this change?

Reply:
I'll add a comment, but this test change proves the need for changing the default context of a Query from context.Background to query.Context because WithContext could have already been called on the query before WrapQuery is called. i.e. the parent trace could get lost depending on the order of calls before this change and the test proves it.